packages/webapp/src/containers/GoogleLoginButton/saga.js

```diff
@@ -17,7 +17,6 @@ export const loginWithGoogle = createAction(`loginWithGoogleSaga`);
 
 export function* loginWithGoogleSaga({ payload: google_id_token }) {
   try {
-    yield put(onLoadingUserFarmsStart());
```

Comment:
Can the fail reducer (and both imports) also be removed then? Are these maybe even just a copy-paste relic since they are called anyway in the farm selection flow?

And I guess I never really understood the relationship between this issue and the Continue button bug because I never even noticed this line here, and getting rid of that intermediate page still fixed the button bug! Did you figure out how they relate in the end?

Reply:
Ohhhhh, that's exactly right!! I did indeed push to `/farm_selection` and not `/welcome` so I unintentionally piggy backed on `getUserFarms` to clean up that loading state! Makes sense.